script/build-simple-js-bundle.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```

Comment:
I believe we already have webpack setup for this cc @bbondy

Reply:
actually the outputs from the brave_sync_lib already exist in the brave-sync and brave-crypto repos in vendor. Do we even need this?

Reply:
see https://github.com/brave/brave-core/pull/226/files#diff-91f51b969eaddd52f16043d734a68bd5R72